src/INode.java

```diff
@@ -431,10 +451,24 @@ private ASTValue evalQueueWait(WordsEnvironment environment) {
 		throw new AssertionError("Not yet implemented");
 	}
 
-	private ASTValue evalReferenceList(WordsEnvironment environment) {
-		// TODO
-		//throw new AssertionError("Not yet implemented");
-		return new ASTValue(ValueType.NOTHING);
+	private ASTValue evalReferenceList(WordsEnvironment environment) throws WordsRuntimeException {
+		String firstObjRefName = children.get(0).eval(environment).stringValue;
+		WordsObject currentObj = environment.getObject(firstObjRefName);
```

Comment:
Right now, getObject can return null, and you'd need to throw an exception from this code in case currentObj is null. I'd be fine with also modifying get object to throw the exception, but we do need to check.